### PR TITLE
Add Alpine glibc images

### PR DESCRIPTION
**Pull request by karianna**
Java 8 JDK and JRE, Java 11 JDK, Java 16 JDK

### CODE REVIEW

**Comment by karianna** [comment]
Apologies but this still needs a discussion/vote from the steering committee (will confirm tomorrow).

**Comment by tianon** [comment]
There's a fair amount of Alpine-related discussion in https://github.com/docker-library/official-images/pull/5710 that should be taken into account as well.

**Comment by gdams** [comment]
> There's a fair amount of Alpine-related discussion in #5710 that should be taken into account as well.

Are you mainly referring to the musl/glibc discussion? We plan on rolling out musl based builds in the near future and will swap the dockerfiles to consume from those binaries in due course. That said, it is unlikely that the musl patches will ever be backported to JDK8 and it could take some time for them to appear in the JDK11 codestream so my preference would be to begin with glibc based images.

**Comment by karianna** [comment]
The Project Management Committee (PMC) is putting this on pause until we get more feedback from our community.  I'll update or withdraw this PR in a week's time or so.

**Comment by karianna** [comment]
Hi all,

**TLDR Decision**

The PMC met today and decided **not to go ahead with the glibc based Alpine images**.

**Reasoning**

The Alpine Committers and Maintainers have made it clear why glibc based Apline Images are a really bad idea (see comments above and/or at the [Adoptium Pull Request](https://github.com/adoptium/containers/issues/1).

The Adoptium PMC believes that as part of distributing a secure and stable supply chain we should adhere to the guidelines from the Alpine maintainers.

**Next Steps**

We will therefore discard this Pull Request from Docker official and remove the subsequent glibc alpine support from the adoptium containers repository.

A blog post explaining this decision and recommending alternatives (e.g. Smaller ubuntu/centos/ubi style images will be made.

**The Future --> Musl Builds**

What we will do is push forward with our musl based alpine image investigations.

Java 16/17 - Those builds are doable today but we need to set up TCK infrastructure for those and see if we can make sure that meets compliance.
Java 8/11 - the various VM Engineers involved in the supporting companies (Azul, Microsoft, Red Hat, IBM, et al) will look to backporting the musl patches to Java 8 and 11. Timeline TBD.
